content/docs/command-reference/exp/run.md

```diff
@@ -183,25 +197,84 @@ experiment we just ran (`exp-44136`).
 `dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
 file before running an experiment.
 
+<admon type="warn">
+
+Note that `--set-param` doesn't update your `dvc.yaml`. If you are tracking each
+parameter individually, make sure to update your `dvc.yaml` accordingly when
+appending or removing parameters.
+
+</admon>
+
+You can optionally provide a prefix `[<filename>:]` in order to edit a specific
+params file. If not provided, `params.yaml` will be used as default.
+
+`<override_pattern>` supports different syntaxes for modifying the parameters:
```

Comment:
Again, I would mention Hydra.

What do you think about the examples? I'm not sure if they are helpful or too repetitive of Hydra docs.

I think we should have an example like `custom_file.json:+newparam=1` because it's easy for me to think it should be `+custom_file.json:newparam=1`.